style.css

```diff
@@ -122,3 +121,306 @@ li img {
   transform: translateY(-20%);
   color: red;
 }
+
+.container {
+  background: #e5e5e5;
+  padding-top: 20%;
+}
+
+.card-container {
+  padding: 6%;
+  font-family: 'Poppins', sans-serif;
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+  font-weight: 700;
+  margin-bottom: 4rem;
+  font-style: normal;
+}
+
+.card {
+  background-color: #fff;
+  padding: 5%;
+  border: 2px solid #dddbdb;
+  border-radius: 15px;
+  font-weight: 900;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+
+  /* padding: 0.5rem 0; */
+}
+
+.primary-text {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+  align-items: flex-start;
+  font-weight: bold;
+}
+
+.tonic {
+  font-family: 'poppins', sans-serif;
+  color: #091e42;
+  font-weight: bolder;
+  font-size: 40px;
+  line-height: 34px;
+  flex-grow: 0;
+  margin-top: 0.5rem;
+}
+
+.frame {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  color: #7a869a;
+}
+
+.heading2 {
+  font-size: 16px;
+  margin-top: -1rem;
+  padding-bottom: 0.5rem;
+  color: #091e42;
+}
+
+.langs li {
+  list-style: none;
+}
+
+.frame ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 4rem;
+  width: 100%;
+}
+
+.tech li {
+  list-style: none;
+  margin-right: 20px;
+  font-size: 12px;
+  background-color: #ebebff;
+  color: #6070ff;
+  font-weight: normal;
+  padding: 2px 10px;
+  border-radius: 7px;
+  margin-bottom: 1rem;
+}
+
+.frame ul li {
+  margin-left: 0.5rem;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.frame .spans {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: #c1c7d0;
+}
+
+.techs {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  padding: 0.3rem;
+}
+
+.img {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+}
+
+.techs ul {
+  list-style: none;
+  display: flex;
+  justify-content: start;
+}
+
+.techs ul li {
+  margin-right: 15px;
+  margin-bottom: 2rem;
+  background-color: #e5e5e5;
+  color: #6070ff;
+  padding: 0.8rem 0.8rem;
+  border-radius: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+}
+
+.tech li:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+/* About styling */
+
+.container-b {
+  font-family: 'Poppins', sans-serif;
+  max-width: 768px;
+  border-radius: 0 20px 0 0;
+}
+
+.card-b {
+  background-color: #fff;
+  padding: 5%;
+  border-radius: 0 60px 0 0;
+}
```

Comment:
- [ ] As mentioned previously the About section should be aligned using Flexbox to give it better responsiveness so kindly add that to your container here. ✔